SPECS/alsa-lib/alsa-lib.spec

```diff
@@ -1,13 +1,17 @@
+%define version_alsa_tplg 1.2.5
+
 Summary:        ALSA library
 Name:           alsa-lib
 Version:        1.2.9
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
```

Comment:
this should change to 
Distribution:   Edge Microvisor Toolkit
Vendor:         Intel Corporation